src/ucp/api/ucp.h

```diff
@@ -3768,19 +3768,17 @@ void ucp_dt_destroy(ucp_datatype_t datatype);
  * When @ref UCP_DATATYPE_ATTR_FIELD_PACKED_SIZE is set in @a field_mask of @a attr,
  * the field @a packed_size is set to the packed size (bytes) of the datatype.
  *
- * @note The size of a generic datatype is available only after it has been used
- *       in a communication operation. If it is queried before,
- *       @ref UCS_ERR_UNSUPPORTED will be returned.
- *
- * @note The size of IOV data cannot be determined from the UCP datatype. If an
- *       IOV @a datatype is passed, @ref UCS_ERR_INVALID_PARAM will be returned.
+ * @note The parameters @a buffer and @a count are ignored for contiguous datatypes.
```

Comment:
IMO this note is redundant

Reply:
Redundant with the description of the `buffer` parameter? Should that go into another PR, together with the API change?

Reply:
We should remove this note because it describes an implementation-defined behavior.
On API layer, we should document only the guarantees, like a specification.